packages/types/src/serviceTypes.ts

```diff
@@ -8,3 +8,13 @@ interface ErrorStatus<E> {
 }
 
 export type Status<T, E> = SuccessStatus<T> | ErrorStatus<E>;
+
+export const mapStatus = <T, E, U>(
```

Comment:
The name seems like it maps the status, but it only maps the data when it succeeds.
What about `mapSuccessData`?

Reply:
I agree – I was a bit unsure, and took direct inspiration from `RemoteData.map`, which only maps the success data, but I agree that it would make more sense to name it `mapSuccessStatus`/`mapSuccessData`/`mapSuccessStatusData` – and a `mapStatusError` might also be nice to have